about.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width initial-scale=1" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link rel="stylesheet" href="./css/stylesmin768.css" />
+    <link rel="stylesheet" href="./css/styles1440.css">
+    <title> Creative Commons Global Summit 2023</title>
+  </head>
+  <body>
+    <header class="header abt-page-bg">
+      <div class="top-menu blur">
+        <ul class="connect">
+            <li class="s-icon1"><a href="#!"><i class="fa-brands fa-facebook"></i></a></li>
+            <li class="s-icon2"><a href="#!"><i class="fa-brands fa-twitter"></i></a></li>
+            <li class="lang"><a href="#!">Eng</a></li>
+            <li class="user-page"><a href="#!">My Page</a></li>
+            <li class="log-in-out"><a href="#!">Logout</a></li>
+       </ul>
+   </div>
+   <!--mobile nav-->
+   <nav class="primary-nav">
+    <!-- logo-->
+      <a href="./index.html" class="logo-container blur"> 
+        <!--logo image-->
+        <img id="logo" src="./images/cc-logo.png" alt="The CC logo">
+      </a> <!--logo ends-->
+      <!--mobile nav - hamburger-->
+      <a href="#" id="main-mobile-menu" class="hamburger-wrapper blur">
+        <img src="./images/mobile-menu.png" alt="Mobile menu icon">
+      </a>
+      <!--mobile menu items -->
+      <input type="checkbox" class="mobile-menu-nav-toggle" id="mobile-menu-nav-toggle">
+      <nav id="mobile-menu-nav" class="mobile-menu-nav">
+          <ul class="mobile-menu">
+              <li><a id="about" class="close-menu" href="./about.html">About</a></li>
+              <li><a id="program"  class="close-menu" href="./index.html">Program</a></li>
+              <li><a id="join"  class="close-menu" href="./index.html">Join</a></li>
+              <li><a id="news" class="close-menu" href="./index.html">News</a></li>
+              <li><a id="mcampaign" class="close-menu" href="./index.html">CC Campaign</a></li>
+            <label for="mobile-menu-nav-toggle" class="mobile-menu-nav-toggle-label">
+              <span>x</span>
+            </label>
+          </ul>
+      </nav>
+        <!--desktop menu-->
+      <ul class="desk-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">News</a></li>
+          <li id="dcampaign" class="campaign-link"><a href="#">CC Campaign</a></li>
+      </ul>
+    </nav> <!--header ends-->
+      <!--headline begins-->
+      <div class="abt-headline blur">
+        <p class="headline-greeting">"Hello! Changing the world"</p>
+        <hr class="headline-hr" />
```

Comment:
[REQUIRED] Kindly put this under the main h1 text just like we have it in Cindy' design